src/data/AxiosHttpClientRepository.ts

```diff
@@ -1,14 +1,14 @@
-import MockAdapter from "axios-mock-adapter";
 import axios, { AxiosInstance } from "axios";
+import MockAdapter from "axios-mock-adapter";
 import qs from "qs";
-
+import { CancelableResponse } from "../repositories/CancelableResponse";
 import {
+    ConstructorOptions,
+    getBody,
```

Comment:
Unused. No body processing for axios repo?

Reply:
Nope, axios auto detects the body and processes it internally ie form data